Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/PyPDF2'...
Listing './lib/python3.5/site-packages/PyPDF2-1.26.0.egg-info'...
Listing './lib/python3.5/site-packages/Unidecode-1.0.22.dist-info'...
Listing './lib/python3.5/site-packages/certifi'...
Listing './lib/python3.5/site-packages/certifi-2017.11.5.dist-info'...
Listing './lib/python3.5/site-packages/chardet'...
Listing './lib/python3.5/site-packages/chardet/cli'...
Listing './lib/python3.5/site-packages/chardet-3.0.4.dist-info'...
Listing './lib/python3.5/site-packages/idna'...
Listing './lib/python3.5/site-packages/idna-2.6.dist-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/python_magic-0.4.15.dist-info'...
Listing './lib/python3.5/site-packages/refextract'...
Listing './lib/python3.5/site-packages/refextract/authors'...
Compiling './lib/python3.5/site-packages/refextract/authors/regexs.py'...
***   File "./lib/python3.5/site-packages/refextract/authors/regexs.py", line 73
    letter_re = re.compile(ur'(\w)', re.U)
                                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/refextract/documents'...
Compiling './lib/python3.5/site-packages/refextract/documents/pdf.py'...
***   File "./lib/python3.5/site-packages/refextract/documents/pdf.py", line 501
    p_break_in_line = re.compile(ur'^\s*\f(.+)$', re.UNICODE)
                                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/refextract/documents/text.py'...
***   File "./lib/python3.5/site-packages/refextract/documents/text.py", line 29
    re_space_comma = re.compile(ur'\s,', re.UNICODE)
                                      ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/refextract/references'...
Compiling './lib/python3.5/site-packages/refextract/references/api.py'...
***   File "./lib/python3.5/site-packages/refextract/references/api.py", line 96
    raise FullTextNotAvailableError("URL not found: '{0}'".format(url)), None, sys.exc_info()[2]
                                                                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/refextract/references/engine.py'...
***   File "./lib/python3.5/site-packages/refextract/references/engine.py", line 175
    re_report = re.compile(ur'^(?P<name>[A-Z-]+)(?P<nums>[\d-]+)$', re.UNICODE)
                                                                 ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/refextract/references/kbs'...
Compiling './lib/python3.5/site-packages/refextract/references/kbs.py'...
***   File "./lib/python3.5/site-packages/refextract/references/kbs.py", line 312
    search_pattern_str = ur'(?:^|[^a-zA-Z0-9\/\.\-])([\[\(]?(?P<categ>' \
                                                                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/refextract/references/regexs.py'...
***   File "./lib/python3.5/site-packages/refextract/references/regexs.py", line 31
    re_sep = ur"\s*[,\s:-]\s*"
                             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/refextract/references/tag.py'...
***   File "./lib/python3.5/site-packages/refextract/references/tag.py", line 345
    return re_isbn.sub(ur'<cds.ISBN>\g<code></cds.ISBN>', line)
                                                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/refextract/references/text.py'...
***   File "./lib/python3.5/site-packages/refextract/references/text.py", line 144
    pattern = ur'\(?\[?(\d{1,4})\]?\)?\.?\s*$'
                                             ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/refextract-0.2.3.egg-info'...
Listing './lib/python3.5/site-packages/requests'...
Listing './lib/python3.5/site-packages/requests-2.18.4.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/six-1.11.0.dist-info'...
Listing './lib/python3.5/site-packages/unidecode'...
Listing './lib/python3.5/site-packages/urllib3'...
Listing './lib/python3.5/site-packages/urllib3/contrib'...
Listing './lib/python3.5/site-packages/urllib3/contrib/_securetransport'...
Listing './lib/python3.5/site-packages/urllib3/packages'...
Listing './lib/python3.5/site-packages/urllib3/packages/backports'...
Listing './lib/python3.5/site-packages/urllib3/packages/ssl_match_hostname'...
Listing './lib/python3.5/site-packages/urllib3/util'...
Listing './lib/python3.5/site-packages/urllib3-1.22.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
